### DISCUSSION TITLE
ddev composer create drupal/recommended-project fails

### DISCUSSION THREAD

**Discussion by ayhankoyun** [ddev/ddev#5268] [General]
Hi,

I try to install Drupal 10. I use Docker Desktop latest version on Windows 10. Everything seems to work fine until the command 'ddev composer create drupal/recommended-project', composer doesn't download the core, at 98% it stucks and did not go any step on. I don't know whats the problem, has had anyone observed same behaviour and can help me?

**Comment by rfay** [comment]
Hi @ayhankoyun - Are you on traditional Windows or WSL2? If traditional Windows, make sure that mutagen is enabled. It's enabled by default on Windows in DDEV v1.22+.

**Reply by ayhankoyun** [reply]
Hi @rfay, thanks for fast reply, I am using WSL2.

**Reply by rfay** [reply]
Please show the full output. Maybe you can attach it as a file, or put it in a gist on gist.github.com

**Reply by ayhankoyun** [reply]
@rfay there is no output, it stands still at 98%.

**Reply by rfay** [reply]
Please show the full output, including the command you're giving. And please say what Docker provider you're using. It would help me a lot if you'd do a `ddev debug test` and put the output in a gist, and also add to the gist the full output of your composer command.

**Reply by ayhankoyun** [reply]
Hey @rfay I tried it again by installing mutagen, now it works fine, Thanks for the hint and help.
